README.md

```diff
@@ -2,21 +2,54 @@
 
-Performant: Using Kubernetes leveraging containers for your services reduces overhead on virtual machines.
+## Why is Zero Reliable, Scalable, Performant and Secure
 
-Security: Properly configured access-control to resources/security groups, using secure vault systems(AWS's secret manager), and following best practices provides great security out of the box.
+Reliability: Your infrastructure will be set up in multiple availability zones making it highly available and fault tolerant. All production workloads will run with multiple instances by default, using AWS ELB and Nginx to load balance traffic. All infrastructure is represented with code using [Hashicorp Terraform][terraform] so your environments are reproducible, auditable, and easy to configure.
+
+Scability: Your services will be running in Kubernetes, with the EKS nodes running in an AWS [Auto Scaling Group][asg]. Both the application workloads and cluster size are ready to scale whenever the need arises. Your frontend assets will be stored in S3 and served from AWS' Cloudfront CDN which operates at global scale.
+
+Security: Properly configured access-control to resources/security groups, using secure vault systems (AWS Secret Manager, Kubernetes secrets), and following best practices provides great security out of the box. Our practices are built on top of multiple security audits and penetration tests. Automatic certificate management using [Let's Encrypt][letsencrypt], database encryption, VPN support, and more mean your traffic will always be encrypted. Built-in application features like user authentication help you bullet-proof your application by using existing, tested tools rather than reinventing the wheel when it comes to features like user management and auth.
+
+
+## What do you get out of the box?
+### Infrastructure
+- Fully configured infrastructure-as-code AWS environment including:
+  - VPCs per environment (staging, production) with pre-configured subnets, security groups, etc.
+  - EKS Kubernetes cluster per environment, pre-configured with tools like cert-manager, external-dns, nginx-ingress-controller
+  - RDS database for your application (Postgres or MySQL)
+  - S3 buckets and Cloudfront distributions to serve your assets
+- Logging and Metrics collected automatically using either Cloudwatch or Prometheus + Grafana, Elasticsearch + Kibana
+- VPN using [Wireguard][wireguard]
+- User management and Identity / Access Proxy using Ory [Kratos][kratos] and [Oathkeeper][oathkeeper] (Optional)
+- Tooling to make it easy to set up secure access for your team
+- Local/Cloud Hybrid developer environment using Skaffold and Telepresence (In Progress)
+
+### Backend
+- Golang or Node example project automatically set up and deployed to your new infrastructure
+- CI pipeline built with [CircleCI][circleci]. Just merge a PR and a deploy will start. Your code will be built and tested, deployed to staging, then prompt you to push to production
```

Comment:
Not sure if its worth it to mention the backend example projects are dockerized so you can ensure the same code you test locally is the same as staging / production. Given we tell them its on k8s it may be self explanatory though. 

and maybe the graceful shutdown might be nice to mention too.